### ISSUE TITLE
Small bug when serializing it to JSON

### ISSUE THREAD

**Issue by daniloefl** [CyberZHG/keras-layer-normalization#1]
Dear @CyberZHG ,

I would like to point out, in case this code is still under use, that this layer is not currently serialisable to JSON. The reason is that one must call `keras.initializers.serialize()` in `get_config`, so that the initializers for the parameters are serialized. This is done for the `BatchNormalization` code, here:

https://github.com/keras-team/keras/blob/master/keras/layers/normalization.py#L210

Best regards,
Danilo

**Comment by CyberZHG** [comment]
Thanks, I've updated and made a release.

**Comment by daniloefl** [comment]
Hi @CyberZHG , thanks. I will close it then.
